src/rust-crypto/libolm_migration.ts

```diff
@@ -380,3 +382,77 @@ async function getAndDecryptCachedSecretKey(
 
     return encodedKey === null ? undefined : await decryptAES(encodedKey, legacyPickleKey, name);
 }
+
+/**
+ * Checks if the legacy store has a trusted public master key.
+ * Returns null if the legacy session was not verified, or the trusted public master key if it was.
+ * @param legacyStore - The legacy store to check.
+ */
+async function maybeTrustedPublicMasterKey(legacyStore: CryptoStore): Promise<string | null> {
```

Comment:
This feels like a slightly odd name? I would feel fine to me for `getTrustedPublicMasterKey` to return null if there isn't one.

Reply:
Ok renamed.